src/sage/manifolds/differentiable/diff_map.py

```diff
@@ -641,11 +641,11 @@ def differential(self, point: ManifoldPoint) -> FiniteRankFreeModuleMorphism:
                   for i in range(n2)]
         bases = (chart1.frame().at(point), chart2.frame().at(image_point))
         if self._name is not None and point._name is not None:
-            name = 'd%s_%s'%(self._name, point._name)
+            name = 'd%s_%s' % (self._name, point._name)
         else:
             name = None
         if self._latex_name is not None and point._latex_name is not None:
-            latex_name = r'{\mathrm{d}%s}_{%s}'%(self._latex_name,
+            latex_name = r'{\mathrm{d}%s}_{%s}' % (self._latex_name,
                                                  point._latex_name)
```

Comment:
I understand you don't want to modify all the code, but we will need another PR for aligning this line.